src/cls/IPM/ResourceProcessor/FileCopy.cls

```diff
@@ -121,7 +132,21 @@ Method NormalizeNames(ByRef pSource As %String, ByRef pTarget As %String, Output
   }
 }
 
-Method DoCopy(tSource, tTarget, pParams)
+Method GetSource()
```

Comment:
@isc-jlechtne 
Nit: I'd name this GetNormalizedSourcePath.
Also, it's worth adding one line of method documentation to explain that this prefixes the path to the module root if no $ variables are used.

Reply:
@isc-eneil I don't want to name it Normalized since the normalization actually happens in the DoCopy method. My comment inside the method is kind of the inverse of what you're saying but changing it to a method description instead as you suggested does make sense.

Reply:
That's fair. I'm good with GetSource then.